cmake/testing.cmake

```diff
@@ -267,6 +267,21 @@ if(F3D_MODULE_OCCT)
   f3d_test(TestIGES spacer.igs "300,300")
 endif()
 
+if(F3D_MODULE_ASSIMP)
+  f3d_test(TestOFF teapot.off "300,300" "--up=+Z")
+  f3d_test(TestDXF PinkEggFromLW.dxf "300,300" "--bg-color=1,1,1 -p")
+
+  if(VTK_VERSION VERSION_GREATER 9.0.20210728)
+    f3d_test(TestFBX RED_BRICK_PILE_MEDUIM_5K.fbx "300,300")
+    f3d_test(TestDAE duck.dae "300,300")
```

Comment:
you can use these files for testing instead:

https://gitlab.kitware.com/f3d/f3d/uploads/8c68ef259733b8c6160a092f04af9f17/untitled.fbx

https://gitlab.kitware.com/f3d/f3d/uploads/5cf1d42cfb742b5a4adcccd2a372e443/untitled.dae

Reply:
These files have textures but the texture files are not included.  
What's wrong with the tests I have added?

Reply:
RED_BRICK_PILE_MEDUIM_5K.fbx is too big and not CC